kratos/solving_strategies/convergencecriterias/residual_criteria.h

```diff
@@ -142,7 +142,7 @@ class ResidualCriteria
         const TSystemVectorType& b
         ) override
     {
-        const SizeType size_b = b.size();
+        const SizeType size_b = TSparseSpace::Size(b);
```

Comment:
I think the same problem exists in the displacement_criteria

Reply:
ping @jcotela